README.md

```diff
@@ -36,6 +36,37 @@ Alternatively, copy the code for the individual macro directly from the .BAS fil
 12. After the macro is saved then it can be mapped to a user tool, key, or key chord as usual (optional).
 Done!
 
+## Setup
+After copying the macrobooks to `C:/Users/[your-name]/AppData/Roaming/OCLC/Connex/Macros/` you can set up your macros in ConnexionClient and map them to user tools.
+
+### Create New User Profile
+When setting up the Connexion Client for the first time, users will need to create a new user profile. Users can do this using the 
```

Comment:
Users can create NYPL and BPL profiles by running the NewUserNYP macro directly.

Reply:
@charlottekostelic @Mfgloger, can you specify this will work only in LSC in CAT (macro pulls data from a network drive, and staff in other departments won't be able to use it - we need to warn them)?